### PR TITLE
Replace REDIRECT_URL with VANITY_URL

### PR DESCRIPTION
**Pull request by ajdeziel**
Fix-a-roo for Document Manager! Replaced `SUFFIX` and `REDIRECT_URL` with `VANITY_URL`.

### CODE REVIEW

**Comment by james-hollinger** [comment]
Need the ephemeral DC file as well

File: openshift/document-manager.dc.yaml [hunk 1]
```diff
@@ -265,7 +265,7 @@ objects:
               key: DOCUMENT_MANAGER_CLIENT_SECRET
               name: "document-manager-secret"
         - name: DOCUMENT_MANAGER_URL
-          value: "https://document-manager${SUFFIX}.pharmanetenrolment.gov.bc.ca/"
+          value: "https://${VANITY_URL}/api/docman/"
```
Comment (james-hollinger) [inline openshift/document-manager.dc.yaml:268]: The document manager does not have a VANITY_URL parameter

Comment (james-hollinger) [reply openshift/document-manager.dc.yaml:268]: (hence why the URL we are seeing in file uploads is https://${VANITY_URL}/api ...)

Comment (poggdogg) [reply openshift/document-manager.dc.yaml:261]: Correct, it needed to be added to the pipeline.  This is done.

**Comment by ajdeziel** [comment]
I've tested PR-741 through the front-end, and was able to submit a site successfully without error. This PR is now ready for review. 👍

**Comment by ajdeziel** [comment]
This work has been merged in as part of #742. This PR will now be closed as a result.

**Pull request closed without merging**